src/gradient.rs

```diff
@@ -167,6 +181,8 @@ impl Gradient {
                 end: end.into(),
             },
             extend: Default::default(),
+            interpolation_cs: ColorSpaceTag::Srgb,
```

Comment:
Maybe? we should have a (non-pub) `const DEFAULT_GRADIENT_SPACE`, in case we want to change this in future. It would be quite bad to miss one of these.

Reply:
I'm okay with this. I'll do that.